exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillStatsTable.java

```diff
@@ -483,7 +483,7 @@ public static ObjectMapper getMapper() {
       Map<StatisticsKind, Object> statisticsValues = new HashMap<>();
       Double ndv = statsProvider.getNdv(fieldName);
       if (ndv != null) {
-        statisticsValues.put(ColumnStatisticsKind.NVD, ndv);
+        statisticsValues.put(ColumnStatisticsKind.NDV, ndv);
```

Comment:
I thought I had fixed the typo in an earlier PR. Not sure why it is still there.

Reply:
I had not rebased to latest master.  Just did and ran all the regression tests successfully.